orchestra/project_api/api.py

```diff
@@ -2,6 +2,7 @@
 from collections import defaultdict
```

Comment:
Such a nice PR! Curiously, what's the effect on performance with respect to number of DB queries?